trappy/ftrace.py

```diff
@@ -124,6 +119,13 @@ def unregister_parser(cls, cobject):
             if cobject == obj:
                 del scope_classes[name]
 
+    def _do_parse(self):
```

Comment:
No, then it becomes a class-private function and you can't call it from inherited classes :) I spent a little while finding that out last night!